### ISSUE TITLE
Highlight changes and zoom

### ISSUE THREAD

**Issue by thibaultmol** [openstreetmap/iD#7145]
At the screen where you can save your changes, having a button that you can click to zoom out so you can see all the changes you've made and also enables the "highlight changes" option would be interesting.
I think it'd be nice becaues that way you can see a clear "what have I done" before you submit

**Comment by quincylvania** [comment]
@thibaultmol I agree this would be cool! We may also consider giving the changes list its own space separate from the save screen, but that's another issue.

**Comment by sun-geo** [comment]
I agree too.
This would maybe also helpful for beginners, to see if they did just accidentally move a node by mistake.
